client/components/post-card/compact.jsx

```diff
@@ -15,43 +15,43 @@ import safeImageUrl from 'lib/safe-image-url';
 import resizeImageUrl from 'lib/resize-image-url';
 import AuthorAndSite from './author-and-site';
 
-export function SmallPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
-	const classes = classnames( 'post-card small', {
+export function CompactPostCard( { translate, post, site, onPostClick = noop, onSiteClick = noop } ) {
```

Comment:
Can we do this as a prop? It's the pattern we use for other components. `<PostCard compact />` which gets the `is-compact` class.